training/app_init.py

```diff
@@ -1,10 +1,21 @@
 from flask import Flask
+from dotenv import load_dotenv
+load_dotenv()
 from flask_bcrypt import Bcrypt
 from training.main import init_db
+from flask_mail import Mail, Message
+import os
 
 app = Flask(__name__)
 from training.controllers.all_controllers import *
 bcrypt = Bcrypt(app)
+app.config['MAIL_PORT'] = os.environ.get("MAIL_PORT")
+app.config['MAIL_USE_SSL'] = False
+app.config['MAIL_USE_TLS'] = False
+app.config['MAIL_DEBUG'] = False
```

Comment:
It would be nice to have all these configured from env variables. You can pass a second argument to `os.environ.get()` to be returned as fallback

Reply:
It will be fixed in the next PR in the next branch.